package.json

```diff
@@ -25,6 +28,7 @@
   "dependencies": {
     "@beyonk/google-fonts-webpack-plugin": "^1.7",
     "@nuxt/image": "^0.7.1",
+    "@nuxtjs/composition-api": "^0.31.0",
```

Comment:
We can't use this version.

Reply:
Changed to `"@nuxtjs/composition-api": "^0.29.3"` version which is the same that sits inside `@vue-storefront/nuxt` package.

We've multiple references in this storefront code to `@nuxtjs/composition-api` package. Beside of that `eslint` reports multiple `import/no-extraneous-dependencies` errors. This is why we need this dependency.